Frontend/voitheia-App/src/app/components/request/request-form/request-form.component.html

```diff
@@ -1,21 +1,20 @@
-<div class="row">
-    <div class="col-md-8 offset-md-2">
-        <h3>{{'Request.Create.Title' | translate }}</h3>
+<div class="row ">
+    <div class="col-md-6 offset-md-3 content-container form-container">
```

Comment:
Why do we have the class "content-container"? If we add it to every page it should be added to the surrounding page instead

Reply:
class content-container removed. Rules now apply globally to all inserted content to keep content to footer relationship constant